docs/product-user-guide/src/main/asciidoc/project-deploying-con.adoc

```diff
@@ -0,0 +1,14 @@
+[id='project-deploying-con_{context}']
+= Deploying a project
+
+The deployment process can vary based on the needs of your infrastructure.
+
+In a simple deployment of {PRODUCT}, there is one {CENTRAL} and one {KIE_SERVER}. You can use {CENTRAL} to develop your business assets and also to manage the {KIE_SERVER}. You can build your project in {CENTRAL} and it is automatically deployed onto the {KIE_SERVER}. To enable automatic deployment, {CENTRAL} includes a built-in Maven repository. You can use {CENTRAL} to manage the {KIE_SERVER}, deploying, removing, starting, and stopping any of the services and their project versions that you have built.
+
+You can also connect several {KIE_SERVERS} to the same {CENTRAL} and group them into different server configurations (in *Menu* -> *Deploy* -> *Execution Servers*). Servers belonging to the same server configuration run the same services, but you can deploy different projects or different versions of projects on different configurations. For example, you could have test servers in the `Test` configuration and production servers in a `Production` configuration. As you develop business assets and services in a project, deploy the project on the `Test` server configuration and then, when a version of the service is sufficiently tested, you can deploy it on the `Production` server configuration.
+
+In this case, to keep developing the service, change the version in the project settings. Then the new version and the old version are seen as different artifacts in the built-in Maven repository. You can deploy the new version on the `Test` server configuration and keep running the old version on the  `Production` server configuration. This deployment process is simple but has significant limitations. Notably, there is not enough access control: a developer can deploy a project directly into production.
+
+If you require a proper integration process, you can use an external Maven repository (for example, Nexus). You can configure {CENTRAL} to push project files into the external repository instead of the built-in repository. You can still use {CENTRAL} to deploy projects on a test {KIE_SERVER}. However, in this process, other {KIE_SERVERS} (for example, staging and production) are not connected to {CENTRAL}. Instead, they retrieve the project KJAR files and any necessary dependencies from your Maven repository. You can progress the KJAR versions through your repository as necessary, in line with the integration process that you want to implement.
+
+If you deploy a {KIE_SERVER} on OpenShift, you can configure it to load and start a service from a Maven repository automatically. If you deploy a {KIE_SERVER} on premises, you can configure access to a Maven repository when setting up the server; then you can use the REST API of the server to load and start a service from the project. A Maven project and a Java client library for automating the API calls are available.
```

Comment:
..on premises... there is that awkward attempt to accurately differentiate from OpenShift :-) Please let me know if you think of anything better.

Reply:
Haha! Truth. This is actually a section from Misha's original version of this. I do think he means everything else but OpenShift,